ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel.hs

```diff
@@ -239,14 +235,10 @@ data ActiveWallet = ActiveWallet {
     }
 
 -- | Initialize the active wallet
-bracketActiveWallet :: MonadMask m
-                    => PassiveWallet
-                    -> WalletDiffusion
-                    -> (ActiveWallet -> m a) -> m a
-bracketActiveWallet walletPassive walletDiffusion =
-    bracket
-      (return ActiveWallet{..})
-      (\_ -> return ())
+activeWalletComponent ::
+       PassiveWallet -> WalletDiffusion -> ComponentM ActiveWallet
+activeWalletComponent walletPassive walletDiffusion =
+    return ActiveWallet{..}
```

Comment:
Why use `return` here instead of `buildComponent_`?

Reply:
I didn't think about it and I think it's not essential now (I can't find an important reason why one should be preferred to another one). One possible answer is "because this function doesn't create any objects (impure objects like `IORef`s or something) and doesn't do anything, so it shouldn't be treated as a component". Anyway, I wouldn't bother until this function actually becomes useful.